scrape/scrape.go

```diff
@@ -128,6 +128,7 @@ func (c *Client) Authenticate(username, password, otpseed string) error {
 		values.Set("password", password)
 	}
 	resp, err := fetchAndSubmitForm(c.Client, "https://github.com/login", setPassword)
+
```

Comment:
Please revert this change.  It is idiomatic Go to tightly couple the handling of an error with the statement that generates the error without a blank line in between.